content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,181 @@ aliases:
+  Existing Kapacitor user credentials should continue to work using the [1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+- Kapacitor Stream-style TICKscripts will not work using the [subscription API](/influxdb/v1.8/administration/subscription-management/).
+  (There is no subscriptions API in InfluxDB 2.0.)
+  We recommend writing data directly to *both* Kapacitor and InfluxDB to allow stream tasks to continue to work.
+  To do this, configure two [InfluxDB output plugins](/telegraf/v1.16/plugins/#influxdb): one for Kapacitor and one for InfluxDB 2.0.
+  If you use other data sources to send data to InfluxDB, Telegraf can be used as an intermediate layer to write to both Kapacitor and InfluxDB.
+  <!-- 
+  Other architectures and mechanisms are possible, too.
+  Need more help? Reach out in the [community forums](https://community.influxdata.com).
+  -->
+
+### User migration
+
+`influxd upgrade` migrates existing 1.x users and their permissions.
+However, it *does not migrate administrative users*.
+
+To review users with admin permissions, in the InfluxDB 1.x CLI, run `show users`.
+Any users labeled "admin" *will not* be migrated.
+
+If you're using an admin user for visualization or Chronograf's administrative functions, you should create a new read-only user before upgrading.
+Admin rights are granted to the primary user created in the InfluxDB 2.0 setup process which runs at the end of the upgrade process.
+(This provides you with the opportunity to re-assess who should be granted admin-level access in your InfluxDB 2.0 setup.)
+
+### Dashboards
+
+You can continue to use your existing dashboards and visualization tools with InfluxDB 2.0 via the [1.x read compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+The upgrade process creates [DBRP mappings](/influxdb/v2.0/reference/api/influxdb-1x/dbrp/) to ensure existing users can execute InfluxQL queries with the appropriate permissions.
+
+However, if your dashboard tool is configured using a user with admin permissions,
+you will need to create a new read-only user with the appropriate database permissions *before* upgrading.
+This new username and password combination should be used within the data source configurations to continue to provide read-only access to the underlying data.
+
+Ensure your dashboards are all functioning before upgrading.
+
+### Other data
+
+The 1.x `_internal` database is not migrated with the `influxd upgrade` command.
+
+### Secure by default
+
+InfluxDB 2.0 no longer supports running with `auth-enabled = false`.
+Authentication is required by default.
+After upgrading, authentication errors can occur if the username and password credentials within InfluxDB do not match what is currently being used by the various applications, agents, visualization tools, etc.
+Consider enabling authentication *before* upgrading to ensure that you have the appropriate credentials in place.
```

Comment:
InfluxDB 2.0 requires authentication and does not support the InfluxDB 1.x `auth-enabled = false` configuration option.
Consider [enabling authentication in your InfluxDB 1.x instance](/influxdb/v1.8/administration/authentication_and_authorization/#set-up-authentication) **before** upgrading to InfluxDB 2.0 to ensure that you have the appropriate credentials in place and that the various applications, agents, and visualization tools are able to connect.